docs/integrations/digitalocean/gslb.mdx

```diff
@@ -0,0 +1,315 @@
+---
+description: In this guide, you'll learn how to layer load balancing between three or more globbly-distributed, cloud-based virtual machines in DigitalOcean.
+---
+
+# Use ngrok's Global Server Load Balancing with DigitalOcean
+
+:::tip TL;DR
+
+To use ngrok's [Global Server Load Balancing](https://ngrok.com/docs/network-edge/gslb/) with DigitalOcean:
+
+1. [Reserve your ngrok domain](#reserve-domain)
+2. [Create your ngrok Edge](#create-edge)
+3. [Create Tunnel Group backends for your VMs](#create-tunnel-group-backends)
+4. [Create a Weighted Backend and Route on your Edge](#create-weighted-backend)
+5. [Install the ngrok agent and an example workload on each VM](#install-ngrok-agent)
+6. [Test out ngrok's Global Server Load Balancing](#test-gslb)
+7. [(optional) Enable the Traffic Policy module for API gateway features](#enable-traffic-policy)
+
+:::
+
+In this guide, you'll learn how to layer ngrok's [Global Server Load Balancing](https://ngrok.com/docs/network-edge/gslb/) (GSLB) on top of a VM-based deployment on DigitalOcean.
+
+GLSB improves the performance and resiliency for your apps by distributing traffic to the nearest Point of Prescence (PoP) and the upstream service. Unlike a traditional GSLB deployment, ngrok automatically routes traffic without requiring you to deploy new infrastructure, provision IPs, or change DNS records. All you have to do is configure your ngrok agents for a simple path to high availability, horizontal scaling, A/B testing, and more.
+
+ngrok's GSLB even works on top of DigitalOcean's internal load balancers to provicde additional global resiliency and latency improvements.
+
+:::caution This how-to guide requires:
+
+1. An [ngrok account](https://ngrok.com/signup).
+1. An account with [DigitalOcean](https://cloud.digitalocean.com/registrations/new).
+1. Three Ubuntu 24.04 virtual machines (VMs) in three globally-distributed regions (e.g. New York, Sydney, and Frankfurt):
+   - Hostnames should be unique, ideally using the location of the datacenter (e.g. `nyc`, `sydney`, and `frankfurt`).
```

Comment:
You may need to change these example locations based on real DO locations. Maybe there's perfect overlap, but it might be confusing if DO doesn't have a Frankfurt location, for example!

Reply:
Those are adjusted locations - Linode has Atlanta, Osaka, and Frankfurt.